draft-ietf-taps-transport-security.md

```diff
@@ -254,57 +257,66 @@ interface or address changes.
 # Security Features
 
 In this section, we enumerate Security Features exposed by protocols discussed in the
-remainder of this document. Protocol security properties that are unrelated to the API surface
-exposed by such protocols, such as client or server identity hiding, are not listed 
+remainder of this document. A Security Feature is a Transport Feature that a network security
```

Comment:
Is a Security Feature really a Transport Feature?

Reply:
Security Features extend the set of Transport Features... (described in ??)

Reply:
Fixed!